rust-src/concordium_base/src/common/types.rs

```diff
@@ -186,6 +187,91 @@ impl Deserial for OwnedParameter {
     }
 }
 
+/// A ratio between two `u64` integers.
+/// It should be safe to assume the denominator is not zero and that numerator
+/// and denominator are coprime.
+#[derive(Debug, SerdeDeserialize, SerdeSerialize, Serial, Clone, Copy)]
+#[serde(try_from = "rust_decimal::Decimal", into = "rust_decimal::Decimal")]
+pub struct Ratio {
```

Comment:
This is moved from the genesis tool, right?

Reply:
Yes, so when this is merged I'll update the PR for genesis-creator